### PR TITLE
Update to lint

### PR DESCRIPTION
**Pull request by orta**
Hah, I tried to use code blocks in my commit title

screen shot 2016-07-17 at 1 25 39 pm

Took ages to figure out why this commit was weird.

Anyway, so I ran through this with `danger plugins lint` as I'm feeling close to getting other plugins on the website now 👍 - wasn't much, I also refactored your `xcov_available` function, in ruby it's custom to make boolean methods finish with a `?`

### CODE REVIEW

File: lib/danger_plugin.rb [hunk 1]
```diff
@@ -65,18 +69,10 @@ def report(*args)
...
     end
 
-    # Class methods
-
-    # Brief description of the plugin
-    # @return  [String]
-    def self.description
-      "Danger plugin to validate the code coverage of the files changed"
-    end
-
```
Comment (orta) [inline lib/danger_plugin.rb:75]: this isn't used anymore 👍

**Comment by nakiostudio** [comment]
Thanks for this! :+1:

**Comment by nakiostudio** [comment]
I bet that one day git will allow that and `:beer:` as part of a commit message :laughing:

**Comment by orta** [comment]
👍  any chance I can get a release with this?

**Comment by nakiostudio** [comment]
Sure thing, it was my plan but I'm on holiday without proper access to a computer :grin: I will try to reach my machine via ssh tonight :+1:

**Comment by orta** [comment]
Great - sounds fine ( no rush, was starting to put together the opening list of plugins, it's not critical )

**Comment by nakiostudio** [comment]
Cool, great to hear that! I've just released 0.1.2 with your changes 👍

**Pull request merged**